style.css

```diff
@@ -2,11 +2,57 @@
 {
   body
   {
-    min-width: 400px;
+    min-width: 900px;
```

Comment:
Some confusion with the widths, add some comments later to specify why min and max are being use, what's being overwritten